examples/siro_sandbox/sandbox_app.py

```diff
@@ -367,6 +365,9 @@ def _reset_environment(self):
                 self._episode_recorder_dict,
             )
 
+        # Call resetting the controller after the app reset since the human action requires to get a human base pos
+        self.ctrl_helper.on_environment_reset()
```

Comment:
Can you update this comment: "hack: we have to reset controllers after AppState reset in case AppState reset overrides the start pose of agents"

In general, this is a bad hack because it's not obvious why this should need to run after AppState reset. We can aim to fix this hack in the future.